internal/provider/transfer_endpoint_metrika.go

```diff
@@ -0,0 +1,154 @@
+package provider
+
+import (
+	"github.com/doublecloud/go-genproto/doublecloud/transfer/v1"
+	"github.com/doublecloud/go-genproto/doublecloud/transfer/v1/endpoint"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type endpointMetrikaSourceSettings struct {
+	CounterIDs     []types.Int64            `tfsdk:"counter_ids"`
+	Token          types.String             `tfsdk:"token"`
+	MetrikaStreams []*endpointMetrikaStream `tfsdk:"metrika_stream"`
```

Comment:
Isn't it better to name it `metrika_streams`?

Reply:
You are right.

Reply:
I will fix it.

Reply:
actually I found it why I removed "s" from metrika_stream because we create metrika source from terraform we will write in this way 
metrika_stream {
     stream_type = "METRIKA_STREAM_TYPE_VISITS"
}
metrika_stream {
     stream_type = "METRIKA_STREAM_TYPE_HITS_V2"
}
so it will be more understanble